test/test_unistd.rs

```diff
@@ -576,6 +577,60 @@ fn test_symlinkat() {
     );
 }
 
+
+#[test]
+#[should_panic]
```

Comment:
You should the `expected("whatever")` string to `should_panic` to ensure that it's only catching the desired panic, and no others.

Reply:
Hi ... Thanks for the tip. Update #3 includes this change.